product_design/how_we_write_user_stories.md

```diff
@@ -1,105 +1,89 @@
-## Why write user stories?
+## Epics and User Stories
 
-Lab Zero writes user stories to place a product’s users at the center of our thinking and decision-making. User stories also support our ability to validate product hypotheses, deliver value early, often and predictably, make better trade-offs, and prioritize effectively.
+* Contents
+{:toc}
 
-**Stories generate interest, empathy and vision.** At the highest level, stories embody the things that make your product worth building — who it’s for and what they can do to achieve their goals. If your team isn’t aligned on these core goals, they won’t know how to prioritize their work to achieve the greatest outcomes for the least effort. 
+## The role of Epics and User Stories in our work 
 
-**Stories focus conversation.** A story allows the team to make multiple confirmations of the task throughout the development process. This begins with agreement on the task to be done, and later, how it will pass acceptance testing.
+Epics and User Stories are tools that focus our conversation around what to build and why. At the highest level, they embody the things that make our product worth building — who it’s for and what that person can do with the product to achieve their goals.
 
-## What is needed before we write user stories?
+- Epics describe the vision and the outcomes we are striving to achieve and how we intend to measure success. 
+- User stories generate empathy and alignment around a discrete effort and how that work will benefit the user. Writing user stories helps us come to an agreement on what needs to be done and what criteria we will use to tell when the work is complete.  
```

Comment:
Should we mention that Epics are composed of a set of stories?  Epics are high level visions or large slices of functionality?  And stories are individual discrete pieces of work in the epic?  (need better word smithing here)

Reply:
@katebennet would you be able to address @Shiun's content questions?